recipes/pango/all/conanfile.py

```diff
@@ -38,16 +38,28 @@ def configure(self):
         del self.settings.compiler.libcxx
         del self.settings.compiler.cppstd
 
+        if(self.options.xft == "auto"):
+            self.options.xft = (self.settings.os == 'Linux' or self.settings.os == 'FreeBSD')
+        if(self.options.fontconfig == "auto"):
+            self.options.fontconfig = (self.settings.os != 'Windows' and self.settings.os != 'Macos')
```

Comment:
I love you C++

to sprinkle a little 🐍 

```suggestion
        if self.options.xft == "auto":
            self.options.xft = self.settings.os in ["Linux", "FreeBSD"]
        if self.options.fontconfig == "auto":
            self.options.fontconfig = not self.settings.os in ["Windows", "Macos"]
```

> ⚠️ typed manually might be a typo

Reply:
Sorry, once a C++ programmer, always a C++ programmer :wink: